codalab_service.py

```diff
@@ -153,7 +153,7 @@ def has_callable_default(self):
     CodalabArg(
         name='version',
         help='Version of CodaLab (usually the branch name)',
-        default=get_default_version(),
+        default="master",
```

Comment:
@teetone the problem was that `get_default_version()` gives an error when run on the tests, because it ends up calling git, but the docker containers don't have git installed. However, I don't think this change is what we want.

@yiboliu -- To avoid calling `get_default_version()` all the time, let's instead do:

```suggestion
        default=None
```

And in the code, later, we can do:

```
version = version or get_default_version()
```

Finally, in test_cli.py, we can do:

```
  _run_command(['codalab-service', '--help', '--version', 'master'], expected_exit_code=0)
```

Reply:
@teetone @epicfaace oh, I actually planned to just hard-code it here and refine it later in another PR, since the work seems not related to this topic. As we discussed last time, I planned to rename the `version` to `branch` to avoid the confusion with the version variable in setup.py and keep using this `get_default_version()` at that time. What's the scenario that we need the user to manually configure the branch name? I think the correct value of the branch can always get retrieved from the function right?

Reply:
I don't think we want to hard-code the version at all, though. Even if it's just temporary, we shouldn't merge a PR to master unless we're okay with releasing it.

The scenario for allowing the user to manually configure the branch name is that if they want to run a particular tag / version of codalab, they can specify it. For example, instead of having to run `git checkout v0.5.2`, to run version 0.5.2, you can just specify `--version v0.5.2` to run it that way.

Reply:
Ok, then that makes sense to me.